tensorrt_llm/_ipc_utils.py

```
@@ -126,15 +142,30 @@ def align_size(size, alignment):
                 )
-                _raise_if_error(error)
+                if error != cudart.cudaError_t.cudaSuccess:
+                    ipc_error = error
+                    continue
                 peer_ptrs.append(ptr)
+                opened_ptrs.append(ptr)
+
+        if not all(dist.tp_allgather(ipc_error is None)):
+            if ipc_error is not None:
+                logger.warning(
+                    f"CUDA IPC is not usable on this system: {repr(ipc_error)}. "
```

Ruff (0.16.0)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)